spec/doublyLinkedList.js

```diff
@@ -0,0 +1,304 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import DoublyLinkedList from '../src/doublyLinkedList'
+
+chai.use( chaiChange )
+
+describe.only( 'DoublyLinkedList', () => {
+  let myDoublyLinkedList
+
+  beforeEach( () => {
+    myDoublyLinkedList = new DoublyLinkedList()
+  })
+
+  it( 'exists', () => {
+    expect( DoublyLinkedList ).to.be.a( 'function' )
+  })
+
+  context( '#getHeadNode()', () => {
+
+    it( 'should return the first node in the Doubly Linked List', () => {
+      myDoublyLinkedList.insert( "apple" )
+      myDoublyLinkedList.insert( "banana" )
+      expect( myDoublyLinkedList.getHeadNode() ).to.have.property( "_data" ).and.to.equal( "apple" )
+      expect( myDoublyLinkedList.getHeadNode().next() ).to.have.property( "_data" ).and.to.equal( "banana" )
+      expect( myDoublyLinkedList.getHeadNode().next() ).to.have.property( "_next" ).and.to.be.null
+    })
+
+    it( 'should return null if Doubly Linked List is empty', () => {
+      expect( myDoublyLinkedList.isEmpty() ).to.be.true
+      expect( myDoublyLinkedList.getHeadNode() ).to.be.null
+    })
+
+  })
+
+  context( '#getTailNode()', () => {
+
+    it( 'should return the last node in the Doubly Linked List', () => {
+      myDoublyLinkedList.insert( "apple" )
+      myDoublyLinkedList.insert( "banana" )
+      expect( myDoublyLinkedList.getTailNode() ).to.have.property( "_data" ).and.to.equal( "banana" )
+      expect( myDoublyLinkedList.getTailNode() ).to.have.property( "_next" ).and.to.be.null
+    })
+
+    it( 'should return null if the Doubly Linked List is empty', () => {
+      expect( myDoublyLinkedList.isEmpty() ).to.be.true
```

Comment:
I would not typically test something unrelated to the test specification - you should be creating a known, empty `list`, and testing the assumption (that `.getTailNode` returns null)